### PR TITLE
Marriage abroad russia sean aisling july 22

### PR DESCRIPTION
**Pull request by stwcontent**
Textual changes to 3 files in the marriage abroad flow for Russia. 

Trello
https://trello.com/c/OKW1xllv/3899-getting-married-abroad-russia-uk-cni-exchange-process-fee

Zen
https://govuk.zendesk.com/agent/tickets/4862149

Working doc:
https://docs.google.com/document/d/1xB_2VZOWq5d1hBmhet3j2wm-qzeX5hd6H83m-0ph-78/edit#

First 3 commits update files individually, changing:

* section on Certificates of No Impediment
* information about payment

Final commit takes in FC amends for all 3 files - see Zen ticket, messages 12-14 July.

### CODE REVIEW

File: app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_other/_opposite_sex.erb [hunk 1]
```diff
@@ -2,22 +2,63 @@ Contact the [Embassy of Russia](/government/publications/foreign-embassies-in-th
...
+
+You must [make an appointment at the British embassy to exchange your CNI](https://www.book-consular-appointment.service.gov.uk/TimeSelection?location=132&service=4).
+
+It costs <%= format_money calculator.consular_fee(:issuing_cni_or_nulla_osta) %> to exchange the CNI.
+
+You can pay using MasterCard or Visa. 
+
+Find out more about [consular fees for Russia](/government/publications/russia-consular-fees).
+
+
```
Comment (MartinJJones) [inline app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_other/_opposite_sex.erb:57]: Small change, please can the extra lines 57 and 58 be removed, thanks

File: app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_local/_opposite_sex.erb [hunk 2]
```diff
@@ -2,22 +2,62 @@ Contact the [Embassy of Russia](/government/publications/foreign-embassies-in-th
...
+
+You must [make an appointment at the British embassy to exchange your CNI](https://www.book-consular-appointment.service.gov.uk/TimeSelection?location=132&service=4).
+
+It costs <%= format_money calculator.consular_fee(:issuing_cni_or_nulla_osta) %> to exchange the CNI.
+
+You can pay using MasterCard or Visa. 
+
+Find out more about [consular fees for Russia](/government/publications/russia-consular-fees).
+
+
```
Comment (MartinJJones) [inline app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_local/_opposite_sex.erb:57]: Small change, please can the extra line 57 be removed, thanks

File: app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_british/_opposite_sex.erb [hunk 3]
```diff
@@ -2,22 +2,62 @@ Contact the [Embassy of Russia](/government/publications/foreign-embassies-in-th
...
+
+You must [make an appointment at the British embassy to exchange your CNI](https://www.book-consular-appointment.service.gov.uk/TimeSelection?location=132&service=4).
+
+It costs <%= format_money calculator.consular_fee(:issuing_cni_or_nulla_osta) %> to exchange the CNI.
+
+You can pay using MasterCard or Visa. 
+
+Find out more about [consular fees for Russia](/government/publications/russia-consular-fees).
+
+
```
Comment (MartinJJones) [inline app/flows/marriage_abroad_flow/outcomes/countries/russia/uk/partner_british/_opposite_sex.erb:57]: Small change, please can the extra line 57 be removed, thanks

**Comment by MartinJJones** [comment]
Nice work @stwcontent , just a couple of small changes, but overall the changes look good to me.

With regards to using the calculator, this looks fine and is used on other outcome pages, for example - https://github.com/alphagov/smart-answers/blob/main/app/flows/marriage_abroad_flow/outcomes/countries/croatia/_same_sex.erb#L55

> Added partial in penultimate commit that pulls in <%= format_money calculator.consular_fee(:issuing_cni_or_nulla_osta) %>
> 
> It looks fine in the deployment, but you may want to make sure I haven't done anything stupid.
> 
> Final commit just restores a sentence from live to the 3 pages.

**Comment by stwcontent** [comment]
@MartinJJones those empty lines should now be gone. This is good to deploy if it looks good to you.

**Review by MartinJJones — APPROVED** [review]
Looks good to me 👍

**Pull request merged**